apps/web/src/common/dialog-controller.tsx

```diff
@@ -227,6 +227,15 @@ export function showMultiDeleteConfirmation(length: number) {
   });
 }
 
+export function showDeleteNotesDialog(type: "Notebook" | "Topic") {
+  return confirm({
+    title: `Delete Contained Notes?`,
+    message: `Do you want to delete notes within this ${type}?`,
+    yesText: `Delete notes`,
+    noText: "No"
+  });
+}
+
```

Comment:
No need to create a common function here for this since it is only being used in 2 places each of which is different.